server/chainquery/queries/claimQueries.js

```diff
@@ -63,19 +63,22 @@ export default (db, table, sequelize) => ({
     });
   },
 
-  getAllChannelClaims: async (channelClaimId) => {
+  getAllChannelClaims: async (channelClaimId, bidState = 'Controlling') => {
```

Comment:
actually, I think this should be either Controlling or Active.

Reply:
@tiger5226 can you confirm?

Reply:
Well they are different bid states. `Controlling` means the claim has won the bid for that name, while `Active` means it has lost. So the question is do they want all claims, regardless of bid state? Then yes, both should be considered. There is also `Accepted`, `Spent` too.

Reply:
I didn't look at the overall code, but judging by the variable name it's looking it up by the claim ID, and not the claim name... So I don't see why we'd want only controlling claims.

Reply:
that's a good point. Basically it will return null if it is not controlling. Seems like this check should be somewhere else more descriptive.

Reply:
As of #668 it's getting Controlling, Active, and Spent:

```
    const whereClause = bidState || {
      [sequelize.Op.or]: [
        { bid_state: 'Controlling' },
        { bid_state: 'Active' },
        { bid_state: 'Accepted' },
      ],
    };
```